### PR TITLE
Shined it up, migrated to React Static

### PR DESCRIPTION
**Pull request by erquhart**

### CODE REVIEW

File: src/App/Footer.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const Footer = ({}) =>
+  <div>
+    <div className="footer">
+      <div className="footer-container">
+        <h3>HeadlessCMS is hosted and maintained by <a href="https://www.netlify.com">Netlify</a>, the perfect way to deploy your JAMstack sites and apps.</h3>
+      </div>
+      <div className="postscript">
+        © Netlify 2017
```
Comment (DirtyF) [inline src/App/Footer.js]: s/2017/2018

Comment (erquhart) [reply src/App/Footer.js]: good catch

**Comment by DirtyF** [comment]
@erquhart curious to know why you moved to react-static (and not another React-based SSG like Gatsby or Phenomic)

**Comment by erquhart** [comment]
Honestly, just wanted to try it out. The value proposition is that it's less opinionated than others, which can be nice for a tiny site like this. So far so good.

**Pull request merged**